.changeset/tricky-moose-sip.md

```diff
@@ -0,0 +1,7 @@
+---
+'default-template': patch
```

Comment:
What's `default-template`?

Reply:
oh yeah woops that should be left out. I think this is from `create-svelte`
```suggestion
```